common/app/views/fragments/nav/headerMenu.scala.html

```diff
@@ -120,8 +120,6 @@
                         </a>
                     </li>
                 }
-
-                @fragments.nav.userAccountLinks()
```

Comment:
Nice! Can we also delete this file? I don't think it is being used by anything else